src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppExtensions.cs

```diff
@@ -281,45 +281,45 @@ public static IResourceBuilder<AzureContainerAppEnvironmentResource> AddAzureCon
             // Exposed so that callers reference the LA workspace in other bicep modules
             infra.Add(new ProvisioningOutput("AZURE_LOG_ANALYTICS_WORKSPACE_NAME", typeof(string))
             {
-                Value = laWorkspace.Name.ToBicepExpression()
+                Value = laWorkspace.Name
```

Comment:
@ArcturusZhang - some times ToBicepExpression doesn't work like I want it to. If the property is set it doesn't always produce the bicep expression, but instead uses the undelrying value. Is there a way to always make the expression?

Reply:
`ToBicepExpression` is meant to do the job of exporting the expressions - but it falls back to values when it cannot.
Currently if you want always expressions, to make sure the only way is to build the expression objects by yourself.
But in the meantime, let me take a look at this test to see why it is not working properly.

Reply:
I looked at this again and I was mistaken. `ToBicepExpression()` is working as expected for all the places I updated in this PR. The generated bicep reads like I expect.